resources/content/schema/admin/admin-auth.json

```diff
@@ -1,46 +1,69 @@
 {
     "authorize" : {
-        "[a-zA-Z]*" : "crud",
-        "[a-zA-Z]*\\..*" : "r",
-
-        ".*\\..*Id" : "cru",
-        ".*\\.data" : "cru",
+    	".*" : "",
```

Comment:
You introduced tab characters here, it looks like. Since you need to fix the wilcard again, fix these tabs by changing them to spaces while you're in there.